tests/feature_activation/test_feature_simulation.py

```diff
@@ -41,46 +47,57 @@
 )
 
 
-class BaseTestFeatureSimulation(SimulatorTestCase):
-    def setUp(self):
-        super().setUp()
-        artifacts = self.simulator.create_artifacts()
-        manager = artifacts.manager
-        manager.allow_mining_without_peers()
-
-        self.feature_service = artifacts.feature_service
-        self.feature_service._feature_settings = _FEATURE_SETTINGS
-        feature_resource = FeatureResource(
-            feature_settings=_FEATURE_SETTINGS,
-            feature_service=self.feature_service,
-            tx_storage=manager.tx_storage
-        )
-        self.web_client = StubSite(feature_resource)
-
-        self.miner = self.simulator.create_miner(manager, hashpower=1e6)
-        self.miner.start()
+class BaseFeatureSimulationTest(SimulatorTestCase):
+    seed_config = 5604003716498505253
```

Comment:
Why are you using a fixed seed?

Reply:
Sorry, this was a rebase artifact. Removed it again in https://github.com/HathorNetwork/hathor-core/pull/645/commits/c050c149f315032b9d068f45113d71d31db4d083

Reply:
Ok!